### DISCUSSION TITLE
GOOGLE APPLICATION CREDENTIALS

### DISCUSSION THREAD

**Discussion by mretcoyy** [imdhemy/laravel-in-app-purchases#179] [Q&A] [answered]
I already set .env GOOGLE_APPLICATION_CREDENTIALS=file_path. but I got 500 error
"Could not load the default credentials. Browse to https://developers.google.com/accounts/docs/application-default-credentials for more information"

can anyone show me how to properly put file path of the credentials.json

**Comment by imdhemy** [comment, marked as answer]
Duplication #140
